### PR TITLE
✨Callout vendor update: adding gdpr_consent and account to Prebid Server urls

### PR DESCRIPTION
**Pull request by bretg**
Adds the gdpr_consent and account parameters to Rubicon and AppNexus Prebid Server RTC vendor entries

### CODE REVIEW

File: extensions/amp-a4a/0.1/callout-vendors.js [hunk 1]
```diff
@@ -60,13 +60,13 @@ const RTC_VENDORS = jsonConfiguration({
   },
   prebidappnexus: {
     url:
-      'https://prebid.adnxs.com/pbs/v1/openrtb2/amp?tag_id=PLACEMENT_ID&w=ATTR(width)&h=ATTR(height)&ow=ATTR(data-override-width)&oh=ATTR(data-override-height)&ms=ATTR(data-multi-size)&slot=ATTR(data-slot)&targeting=TGT&curl=CANONICAL_URL&timeout=TIMEOUT&adcid=ADCID&purl=HREF',
+      'https://prebid.adnxs.com/pbs/v1/openrtb2/amp?tag_id=PLACEMENT_ID&w=ATTR(width)&h=ATTR(height)&ow=ATTR(data-override-width)&oh=ATTR(data-override-height)&ms=ATTR(data-multi-size)&slot=ATTR(data-slot)&targeting=TGT&curl=CANONICAL_URL&timeout=TIMEOUT&adcid=ADCID&purl=HREF&gdpr_consent=ATTR(consentString)',
```
Comment (zhouyx) [inline extensions/amp-a4a/0.1/callout-vendors.js]: The macro is `CONSENT_STRING`. 
Please also include the `CONSENT_STRING` to the `macros` declaration below. Thanks. 
You can refer the usage here for an example. 
https://github.com/ampproject/amphtml/blob/5309266d912717789127b931ff73c6d00c835cee/extensions/amp-a4a/0.1/callout-vendors.js#L116-L128

Comment (bretg) [reply extensions/amp-a4a/0.1/callout-vendors.js]: Thanks - fixed.

**Comment by bretg** [comment]
Working on getting Rubicon to sign the CLA.

**Comment by bretg** [comment]
Tests succeeded on Chrome, Safari, IE, Firefox, failed on Windows 10. Since I just added a parameter to the end of a URL, am suspecting that it's a test environment issue like we experience over on Prebid sometimes. Once I get the CLA issue sorted out, will ask for a re-build.

**Comment by zhouyx** [comment]
Thanks @bretg Let me know when the CLA is ready. Looks like a flaky test, I've tried rebuild.

**Comment by bretg** [comment]
@googlebot I signed it!

**Comment by bretg** [comment]
@googlebot I signed it!

**Comment by bretg** [comment]
@googlebot I signed it!

**Comment by bretg** [comment]
Ok @zhouyx - CLA's done. Note that I also added 'ACCOUNT' to the URL, so am requesting re-review.
Looking for guidance on how to resolve the ampproject/owners-check and percy/amphtml checks. Thanks.

**Comment by zhouyx** [comment]
Looks great. PR merged!

**Comment by bretg** [comment]
thanks @zhouyx 

Question about AMP GDPR. The other Prebid interfaces accept an optional GDPR in-scope flag from the CMP. Does the AMP CMP have the ability to pass whether the current user is considered in-scope for GDPR?

https://iabtechlab.com/wp-content/uploads/2018/02/OpenRTB_Advisory_GDPR_2018-02.pdf
```
OpenRTB v2.2 - v2.5

An exchange must determine whether or not it believes that GDPR is in effect for a given auction. This
may be via a signal by the publisher who has come to this conclusion, detecting that the user is located in an EU nation, or by some other means. Please consult official GDPR resources or legal counsel for making this determination.

The “Regs” object will signal whether or not the request is subject to GDPR regulations. It will do so via the extension attribute “gdpr” which is an optional integer that indicates: 0 = No, 1 = Yes. Under
OpenRTB conventions for optional attributes, omission indicates Unknown.

                      Regs.ext.gdpr
```
I don't believe this is the same as AMP's `consentStateValue`, which is the user's disposition if I understand it correctly. Rather, `regs.ext.gdpr` is a flag letting downstream parties know that the current request is from the European Economic Area or that the publisher wants it to be treated as such.

**Comment by zhouyx** [comment]
Thanks for the feedback. 

I am assuming that in the case where a consent_string is presented, the CMP indicates that the user is in the EEA. If not, the CMP has the full control of the consent string, is it common use case to include the bit to the head of the consent string? 

As you mentioned, consentStateValue is not designed for this use case. Could you please give me an example? If this is a common use case, we are always open to introduce a new variable to indicate GDPR in-scope flag. Thanks.

**Comment by adamleslie** [comment]
Hi @zhouyx ,

Trust you're well mate.

When is this due for release? - we're helping confirm this is working as expected in PROD.

Cheers,
Adam

**Comment by zhouyx** [comment]
Hello Adam, Thank you for reaching out. 

The change shipped with this release https://github.com/ampproject/amphtml/releases. 
Just verified that it's in pre-release canary. You can test it by opt into our dev channel [here](https://cdn.ampproject.org/experiments.html)

Regarding the release date, it is always one week after the change lives in canary. 
Thanks

**Pull request merged**